use_cases/examples/sudden_appearance_new_dns_domain.conf

```diff
@@ -0,0 +1,25 @@
+##############################
+# SUDDEN APPEARANCE - NEW DNS DOMAIN
+##############################
+[sudden_appearance_new_dns_domain]
+es_query_filter=_exists_: Company
+
+target=Company
+aggregator=host, host
```

Comment:
confusing that this is twice the same field, I guess it's an error?
The exact behavior ofmultiple aggregators in this model should also be carefully explained: does the combo of both field values have to be unique then in order to trigger?

Reply:
> confusing that this is twice the same field, I guess it's an error?

Yes that's an error. Was there for test purpose.

> The exact behavior of multiple aggregators in this model should also be carefully explained: does the combo of both field values have to be unique then in order to trigger?

Yes the combo has to be unique.
The behavior of aggregator parameter is similar to the other models.
The definition [here](https://github.com/NVISO-BE/ee-outliers/blob/dev-sudden-appearance-analyzer/documentation/CONFIG_PARAMETERS.md#sudden-appearance-parameters) is not clear enough?

Reply:
It's not very clear in the documentation that the combination of several fields will be used in case the aggregator has mulitple fields, so maybe make that a bit more clear.